src/components/content/order/migrate/ImportServiceData.tsx

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+export const ImportServiceData = (): JSX.Element => {
+    const importDataContentDescription: string = 'The import function is temporarily unavailable.';
```

Comment:
`The data import function is not yet implemented`

Reply:
accept